Piwik.Tracker/PiwikTracker.cs

```diff
@@ -397,6 +397,15 @@ public string GetCustomTrackingParameter(string trackingApiParameter)
         }
 
         /// <summary>
+        /// Gets the stored tracking actions, that have been stored for bulkRequest <see cref="EnableBulkTracking"/>, <see cref="DoBulkTrack"/>.
+        /// </summary>
+        /// <returns></returns>
+        public string[] GetStoredTrackingActions()

```

Comment:
From side of testing you are right. But from side of consumer this method is needed because DoBulkTrack() throws if no requests are stored and the consumer is not able check how many requests are stored. Maybe this should have been a seperate PR therefore. What do you think?